spec/compiler/semantic/while_spec.cr

```diff
@@ -171,7 +171,7 @@ describe "Semantic: while" do
       )) { int32 }
   end
 
-  it "restricts type after while with not and and (#4242)" do
+  it "restricts type after `while` with `not` and `and` (#4242)" do
```

Comment:
I think that `it` should be ok w/ this notation, so if that's ok w/ everyone, this seems to be the easiest way to disambiguate an English _and_ from the token `and`.

Reply:
Yes, I think that is the best notation for this.